services_app/src/androidTest/java/org/opendatakit/utilities/AbstractPermissionsTestCase.java

```diff
@@ -1410,13 +1414,9 @@ private boolean ableToChangeRow(String rowId, boolean isTableLocked, boolean pri
         return false;
       }
 
-      if ( changeServerPrivilegedMetadata && localRowDefaultAccessValue == serverRowDefaultAccessValue ) {
```

Comment:
While your change is smaller, it collapses many ideas into one concept. Sometimes it better to have the concepts you are testing for to be separate. As code changes we need to change the tests and sometimes having simple checks makes it easier to understand what is being tested. Plus the comment no longer matches well.

Reply:
Okay. Thanks for the pointer. I have made the changes.